test/webserv/integration/test_connection.py

```
@@ -153,3 +166,105 @@ def test_timeout_no_request_disconnect() -> None:
+        # GET .unknownデータしてContent-Typeをassert
+        con.request("GET", "/" + UPLOAD_DIR + UNKNOWN_FILENAME)
+        response = con.getresponse()
+
+        assert_status_line(response, HTTPStatus.OK)
+        # 不明なものは"application/octet-stream"が良いらしい？がwebservはdefaultをtext/plainとしている
+        assert_header(response, CONTENT_TYPE, TEXT_PLAIN)
+        assert response.read() == UNKNOWN_DATA
+
+    except HTTPException as e:
+        print(f"Request failed: {e}")
+        raise AssertionError
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)